bot/cogs/snekbox.py

```diff
@@ -177,7 +199,8 @@ def get_status_emoji(results: dict) -> str:
     @command(name="eval", aliases=("e",))
     @guild_only()
     @in_channel(Channels.bot, hidden_channels=(Channels.esoteric,), bypass_roles=EVAL_ROLES)
-    async def eval_command(self, ctx: Context, *, code: str = None) -> None:
+    async def eval_command(self, ctx: Context, action: Optional[Literal["-save", "-load"]] = None,
+                           *, code: str = None) -> None:
```

Comment:
This does not conform to typical project style. Multiline arguments are dropped after the brackets:
```py
    async def eval_command(
        self, ctx: Context, action: Optional[Literal["-save", "-load"]] = None, *, code: str = None
    ) -> None:
```

Reply:
Doesn't this fprmat violate PEP8?

Reply:
No.
> The closing brace/bracket/parenthesis on multi-line constructs may either line up under the first non-whitespace character of the last line of li